apps/src/feedback.js

```diff
@@ -883,13 +886,7 @@ FeedbackUtils.prototype.getGeneratedCodeElement_ = function(options) {
       options.generatedCodeDescription);
   var code = this.studioApp_.polishGeneratedCodeString(this.getGeneratedCodeString_());
 
-  var codeDiv = document.createElement('div');
-  codeDiv.innerHTML = require('./templates/code.html.ejs')({
-    message: infoMessage,
-    code: code
-  });
-
-  return codeDiv;
+  return (<Code message={infoMessage} code={code} />);
 };
```

Comment:
Actually, I wonder if it would make the most sense to have this method return our message and code, and then have the caller create the React component. I think it will be less confusing if we limit the number of places we're passing around React components.